### ISSUE TITLE
Void softlock

### ISSUE THREAD

**Issue by Harmiess-01** [Ciastex/ArcadeCoinBugCentral#162]
#### Expected behavior
The player shouldn't be able to softlock themselves.
   
#### Actual behavior
The player is able to softlock themselves within the void level
  
#### Steps to reproduce
1. Enter the void level
2. Get to the black and orange structure's folding platform at the end of the parkour
3. Pass the trigger by hugging the outside wall (happens on both sides, I just demonstrated it on the left twice)
  
#### Additional information
- This can only be fixed by re-entering the map

#### Videos
![gttod_void](https://user-images.githubusercontent.com/40811168/143728571-78dd024a-d96e-4352-9c15-e532a88e3ebc.gif)
Overshooting the platform
![gttod_void_platform](https://user-images.githubusercontent.com/40811168/143728718-2494d6ad-9fc7-46c7-a5a9-ac92d3823faf.gif)
Triggered after landing

**Comment by Frontrider** [comment]
Exactly. That is just what happened to me because I built up too much momentum and was punished for trying to save myself.

**Comment by andrade0cabelo** [comment]
same issue here, really annoying having to restart the run.
